### DISCUSSION TITLE
Asteroid 7482 (1994) PC1

### DISCUSSION THREAD

**Discussion by Mech8080** [Stellarium/stellarium#2170] [General]
Hello!

I'm trying to locate Asteroid 7482 (1994) PC1 but it does not appear when I type it into the search window. I've updated the star catalogs  but still can't find it.  Am I missing an update or is there a setting I need to click? Thanks!

**Comment by gzotti** [comment]
Updating the star catalogs won't load new asteroids. Chapter 12.7 in the User Guide explains what to do.

**Reply by Mech8080** [reply]
Thanks! Found it!
